resources/zib/zib-TreatmentObjective.xml

```diff
@@ -0,0 +1,199 @@
+      </mapping>
+    </element>
+    <element id="Goal.lifecycleStatus">
+      <path value="Goal.lifecycleStatus" />
+      <comment value="No zib concept can be mapped (implicitly). Unless `lifecycleStatus` is explicitly recorded, it is expected that only _active_ goals are exchanged." />
+    </element>
+    <element id="Goal.description.text">
+      <path value="Goal.description.text" />
+      <short value="DesiredHealthcareResult / StatusName / StatusValue" />
+      <definition value="The textual account of the treatment objective. This concept provides the option to define a treatment objective which cannot be described as a measurement value or functional/mental status." />
+      <comment value="If the concept DesiredHealthcareResult (NL-CM:13.5.5) is not present in this instance of zib TreatmentObject, but DesiredHealthCondition (NL-CM:13.5.3) is, `description.text` should be filled with a textual representation of FunctionalOrMentalStatus.StatusName and FunctionalOrMentalStatus.StatusValue." />
+      <alias value="GewenstZorgresultaat" />
+      <mapping>
+        <identity value="zib-treatmentobjective-v3.2-2020EN" />
+        <map value="NL-CM:13.5.5" />
+        <comment value="DesiredHealthcareResult" />
+      </mapping>
+      <mapping>
+        <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+        <map value="NL-CM:4.26.2" />
+        <comment value="StatusName (implicit, main mapping is on Goal.target.measure)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-functionalormentalstatus-v3.2-2020EN" />
+        <map value="NL-CM:4.26.3" />
+        <comment value="StatusValue (implicit, main mapping is on Goal.target.detailCodeableConcept)" />
+      </mapping>
+    </element>
+    <element id="Goal.target">
+      <path value="Goal.target" />
+      <short value="DesiredHealthCondition" />
+      <definition value="The desired health condition is a target value which can be expressed as the patient’s functional or mental status." />
+      <alias value="GewensteGezondheidstoestand" />
+      <max value="1" />
+      <mapping>
+        <identity value="zib-treatmentobjective-v3.2-2020EN" />
+        <map value="NL-CM:13.5.3" />
+        <comment value="DesiredHealthCondition" />
+      </mapping>
+    </element>
+    <element id="Goal.target.extension">
+      <path value="Goal.target.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="Goal.target.extension:medicalDevice">
+      <path value="Goal.target.extension" />
+      <sliceName value="medicalDevice" />
+      <min value="0" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-FunctionalOrMentalStatus.MedicalDevice" />

```

Comment:
Agree that this should be an ext-TreatmentObject.xxx extension. My recommendation would be to let it be ext-TreatmentObject.MedicalDevice extension.

Reply:
I think you mean ext-TreatmentObjective.MedicalDevice?

Reply:
Indeed!